addons/map/functions/fnc_flashlightGlow.sqf

```diff
@@ -0,0 +1,41 @@
+/*
+ * Author: voiper
+ * Add or remove global flashlight glow for when player is looking at map.
+ *
+ * Arguments:
+ * 0: Flashlight classname ("" for off) <STRING>
+ *
+ * Return value:
+ * None
+ *
+ * Example:
+ * ["ACE_Flashlight_MX991"] call ace_map_fnc_flashlightGlow;
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params ["_flashlight"];
+
+_light = GVAR(glow);
+if (!isNull _light) then {deleteVehicle _light};
+
+if (_flashlight != "") then {
+    _colour = getText (configFile >> "CfgWeapons" >> _flashlight >> "ItemInfo" >> "FlashLight" >> "ACE_Flashlight_Colour");
+
+    _class = switch (_colour) do {
```

Comment:
1. I'd suggest adding either a default value or some error handling in case someone uses the wrong color input.
2. Switch is case sensitive. Maybe add an `toLower` here?